### ISSUE TITLE
Roadmap and use of wiki

### ISSUE THREAD

**Issue by estella144** [estella144/wikidebat#3]
@dzkwsk we need a roadmap for completion. On 3 March 2021, you said:

> Would this summer, in July or August, be acceptable against your process?

Yep (💯/100). But we need a proper roadmap here, not just a deadline. We can put it in the Wiki section.

**Comment by dzkwsk** [comment]
Yes, I will prepare this. I'll put the dots in the wiki, and maybe I can
open a trello for that?
It's cool to meet people for this project. I think it is not possible on
its own and neither is it interesting.

By the way, my name is Christophe Dzikowski, I am a statistician, in France.

 I'll keep you posted for the roadmap soon in the wiki and in two weeks, I
think, for the ontology.

Christophe

**Comment by estella144** [comment]
Ok. Firstly, French? That might explain why you were commenting at 21:01.
@dzkwsk I go by my first name only.
Sounds good to me. I will keep you updated 👍 so that we can get the 🥇 !

Oliver
Mon 22 March, 2021 at 06:39 (UTC)

**Comment by estella144** [comment]
@dzkwsk what's an ontology? **English, please.**

**Comment by estella144** [comment]
@dzkwsk aren't you going to get that roadmap ready?
